[2025-02-05T18:02:32.787-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-05T18:02:32.805-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-05T11:00:00+00:00 [queued]>
[2025-02-05T18:02:32.820-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-05T11:00:00+00:00 [queued]>
[2025-02-05T18:02:32.822-0500] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-02-05T18:02:32.845-0500] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): bot_ex_bbva> on 2025-02-05 11:00:00+00:00
[2025-02-05T18:02:32.852-0500] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8437) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-05T18:02:32.854-0500] {standard_task_runner.py:72} INFO - Started process 8438 to run task
[2025-02-05T18:02:32.854-0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_reporte_como_vamos', 'bot_ex_bbva', 'scheduled__2025-02-05T11:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/etl_como_vamos_dag.py', '--cfg-path', '/tmp/tmplb57vj41']
[2025-02-05T18:02:32.856-0500] {standard_task_runner.py:105} INFO - Job 59: Subtask bot_ex_bbva
[2025-02-05T18:02:32.915-0500] {task_command.py:467} INFO - Running <TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-05T11:00:00+00:00 [running]> on host 21f1c1b48730
[2025-02-05T18:02:33.016-0500] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_reporte_como_vamos' AIRFLOW_CTX_TASK_ID='bot_ex_bbva' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-05T11:00:00+00:00'
[2025-02-05T18:02:33.018-0500] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-05T18:02:33.037-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-05T18:02:33.039-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/bot_etl_ex_bbva.py']
[2025-02-05T18:02:33.049-0500] {subprocess.py:99} INFO - Output:
[2025-02-05T18:02:33.932-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:33,931 - bot_etl_ex_bbva - INFO - Bot ETL Exito-BBVA started
[2025-02-05T18:02:34.472-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:34,472 - bot_etl_ex_bbva - INFO - Conexi√≥n al Selenium Hub establecida correctamente
[2025-02-05T18:02:36.421-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:36,421 - bot_etl_ex_bbva - INFO - Credentials input success
[2025-02-05T18:02:42.463-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:42,463 - bot_etl_ex_bbva - INFO - Click in login button
[2025-02-05T18:02:52.484-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:52,484 - bot_etl_ex_bbva - INFO - Loging success
[2025-02-05T18:02:53.576-0500] {subprocess.py:106} INFO - 2025-02-05 18:02:53,576 - bot_etl_ex_bbva - INFO - Input file succes
[2025-02-05T18:03:04.412-0500] {subprocess.py:106} INFO - 2025-02-05 18:03:04,409 - bot_etl_ex_bbva - INFO - Click in Cargar
[2025-02-05T18:03:10.083-0500] {subprocess.py:106} INFO - 2025-02-05 18:03:10,083 - bot_etl_ex_bbva - INFO - bot_etl_ex_bbva finish execution
[2025-02-05T18:03:10.247-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-05T18:03:10.307-0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-05T18:03:10.309-0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_reporte_como_vamos, task_id=bot_ex_bbva, run_id=scheduled__2025-02-05T11:00:00+00:00, execution_date=20250205T110000, start_date=20250205T230232, end_date=20250205T230310
[2025-02-05T18:03:10.342-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-05T18:03:10.381-0500] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T18:03:10.386-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
